frontend/src/assets/css/typography.scss

```diff
@@ -0,0 +1,82 @@
+@import url("https://fonts.googleapis.com/css2?family=Plus+Jakarta+Sans:wght@400;600;700;800&display=swap");
+
+:root {
+  --font-family: "Plus Jakarta Sans";
+  --font-weight-regular: 400;
+  --font-weight-semibold: 600;
+  --font-weight-bold: 700;
+  --font-weight-extrabold: 800;
+  --line-height-normal: normal;
+}
+
+.h2,
+.h3,
+.h4,
+.h5,
+.h6,
+.main-bold-tall,
+.main-bold,
+.main-regular,
+.main-regular-uppercase,
+.additional {
+  color: var(--general-dark, #222222);
+  font-family: var(--font-family);
+  line-height: normal;
+}
+
+.h2,
+.h3,
+.h4,
+.h5 {
+  font-weight: var(--font-weight-bold);
+}
+
+.h2 {
+  font-size: 42px;
+  text-transform: capitalize;
+}
+
+.h3 {
+  font-size: 28px;
+  line-height: 1.3;
+}
+
+.h4 {
+  font-size: 24px;
+}
+
+.h5 {
+  font-size: 18px;
+}
+
+.h6 {
+  font-weight: var(--font-weight-semibold);
+  font-size: 16px;
+  line-height: 1.405;
+}
+
+.main-bold-tall {
+  font-weight: var(--font-weight-regular);
+  font-size: 18px;
+}
+
+.main-bold {
+  font-weight: var(--font-weight-regular);
+  font-size: 14px;
+}
+
+.main-regular {
+  font-weight: var(--font-weight-semibold);
+  font-size: 16px;
+}
+
+.main-regular-uppercase {
+  font-weight: var(--font-weight-extrabold);
+  font-size: 16px;
+  text-transform: uppercase;
```

Comment:
Create the `.uppercase { text-transform: uppercase; }` for such cases

Reply:
Done.